plasma_framework/migrations/2_deploy_plasma_framework.js

```diff
@@ -19,4 +21,6 @@ module.exports = async (
 
     const plasmaFramework = await PlasmaFramework.deployed();
     await plasmaFramework.activateChildChain({ from: authorityAddress });
+    const sha = childProcess.execSync('git rev-parse HEAD').toString().trim().substring(0, 7);
+    await plasmaFramework.setVersion(''.concat(pck.version, '+', sha), { from: maintainerAddress });
```

Comment:
@boolafish how would I check if this transaction succeeded? currently I think there's nothing checking if this actually worked (same for `activateChildChain`).

Reply:
hmmm, if it reverts, it would stop the migration script just like throwing errors without catching. 
If you want to assert probably we can use the node js library here to check with the result from your `getVersion` result.

https://nodejs.org/api/assert.html